package.json

```diff
@@ -46,5 +46,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "prettier":{
+      "trailingComma": "es5",
+      "tabWidth": 4,
+      "semi": false
```

Comment:
this is for adding semicolons to the end of all lines, right? true karna better rahega imo